### DISCUSSION TITLE
what questions would you like to ask the core team about their experience designing nu shell?

### DISCUSSION THREAD

**Discussion by ghost** [nushell/nushell#3664] [General]
## background and this discussion's goal
I have been wondering about what designing the nu shell has been like from the perspective of the core team.
I wanted to propose a few questions here and see if you had any too!
The goal would be to gather these into a rough outline that could then be used in a panel style discussion amongst the core team.

## a few of my questions

- what other shell designs have influenced your decisions when architecting nu shell?
- what is it like to design a shell / programming language in the open?
- how does the design process in the nu projects and community compare with your experience in software development as a whole?

**Comment by stevenbarragan** [comment]
* What features, current or up coming, are you exited about the most?
* is nu shell a programing language?
* What's the future of nu shell?

**Comment by ghost** [comment]
- How do you feel about the nu project and emerging community?
- How do you balance working on nu with the other demands of your daily life?
